src/partials/gallery.html

```diff
@@ -0,0 +1,6 @@
+<section class="section-gallery">
+    <div class="container">
```

Comment:
Zmień proszę nazwę klasy na `gallery__container`

W katalogu sass utwórz partiala `_gallery.scss` i w treści daj:
@use './variables' as var;
@use './global';

.gallery {
  &__container {
    @extend .container;
}
}

Pozwoli nam to na większą elastyczność jakbyśmy zdecydowali się przebudowywać galerię.